XenAdmin/Wizards/PatchingWizard/PatchingWizard_PatchingPage.cs

```diff
@@ -207,29 +205,6 @@ private List<HostPlan> CompileManualHostPlan(Pool pool, out List<Host> applicabl
                             hostplans.Add(new HostPlan(server, null, updateActions, null));
                         }
                 }
-                else if (SuppPackVdis != null) //supp pack
-                {
-                    foreach (var server in SelectedServers)
-                        if (SuppPackVdis.ContainsKey(server) && poolHosts.Contains(server))
-                        {
-                            var updateActions = new List<PlanAction> {new InstallSupplementalPackPlanAction(server, SuppPackVdis[server])};
-                            hostplans.Add(new HostPlan(server, null, updateActions, null));
-                        }
-                }
-            }
-            else // legacy
-            {
-                foreach (var server in SelectedServers)
-                    if (poolHosts.Contains(server))
-                    {
-                        var hostRebootRequired = WizardHelpers.IsHostRebootRequiredForUpdate(server, Patch, LivePatchCodesByHost);
-
-                        var updateActions = new List<PlanAction> { new ApplyPatchPlanAction(server, Patch, hostRebootRequired) };
-                        hostplans.Add(new HostPlan(server, null, updateActions, null));
-                    }
-
-                if (RemoveUpdateFile && hostplans.Count > 0)
-                    hostplans[hostplans.Count - 1].UpdatesPlanActions.Add(new RemoveUpdateFile(pool, Patch));
             }

```

Comment:
Ok, `UpdateType` can go completely, but I'll do it in a different PR because the changes in PatchingWizard_ModePage overlap with the guidance changes. For now I'll commit the rest of the corrections.